### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by dibdas**
A calculator app where you can calculate the 4 basic Math operations for a company called Math Magicians.
 Built with
- React.js

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUESTED 🔴 ♻️ 
Hello @dibdas 👋🏾 👋🏾 congratulations on making it this far 😄 . 

#### Highlights

- You have hosted your app✔️
- You have integrated linter checks ✔️  

#### To improve
- [x] Ensure that unnecessary files or assets, such as the `React svg Logo`, are deleted ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

File: public/index.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <!--
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (afanimartin) [inline public/index.html]: - Delete comments in the code. Also, delete the `./public/favicon.ico` file. This is a react logo. You don't need it.

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested change. 
Let's remove the `favicon.ico` file. This is a react logo which we don't need.


### Martin Afani
#### Slack: whiz25

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**